README.md

```diff
@@ -1365,7 +1365,71 @@ transportation=CyclingPublicTransport(
   boarding_time=300,
   max_changes=MaxChanges(enabled=True, limit=3)
 )
-``` 
+```
+
+### Proto Transportation
+
+Proto requests transportation param is a union of 3 types.
+
+transportation: Union[ProtoTransportation, PublicTransportWithDetails, DrivingAndPublicTransportWithDetails]
+
+Examples:
+
+#### ProtoTransportation
+
+Select transportation mode, no extra parameters
+
+```python
+from traveltimepy.dto.requests.time_filter_proto import ProtoTransportation
+
+transportation=ProtoTransportation.PUBLIC_TRANSPORT
+
+transportation=ProtoTransportation.DRIVING 
+
+transportation=ProtoTransportation.DRIVING_AND_PUBLIC_TRANSPORT 
+
+transportation=ProtoTransportation.DRIVING_FERRY 
+
+transportation=ProtoTransportation.WALKING 
+
+transportation=ProtoTransportation.CYCLING 
+
+transportation=ProtoTransportation.CYCLING_FERRY 
+
+transportation=ProtoTransportation.WALKING_FERRY 
+```
+
+#### PublicTransportWithDetails
+
+Uses `ProtoTransportation.PUBLIC_TRANSPORT`, allows setting `walking_time_to_station`.
```

Comment:
PublicTransport transporation mode allows to set `walking_time_to_station` which limits the possible duration of walking paths `walking_time_to_station` limit is of low precedence and will not override the global travel time limit